studio/src/app/utils/editor/create-slides.utils.tsx

```diff
@@ -94,7 +96,7 @@ export class CreateSlidesUtils {
 
             const end = <section slot="end"></section>;
 
-            const slide: JSX.IntrinsicElements = <deckgo-slide-split key={uuid()}>
+            const slide: JSX.IntrinsicElements = <deckgo-slide-split vertical={vertical} key={uuid()}>
```

Comment:
Should be replaced by 

```
const slide: JSX.IntrinsicElements = <deckgo-slide-split key={uuid()} {...attributes}>
``

Reply:
I get the following error log:
```log
Type '{ style?: string; src?: string; customBackground?: string; imgSrc?: string; imgAlt?: string; content?: string; customQRCode?: boolean; type?: SlideChartType; innerRadius?: number; animation?: boolean; datePattern?: string; ... 7 more ...; key: string; }' is not assignable to type 'DeckgoSlideSplit'.
  Types of property ''customBackground'' are incompatible.
    Type 'string' is not assignable to type 'boolean'.
```

Reply:
Add `// @ts-ignore` before it

```
// @ts-ignore
const slide: JSX.IntrinsicElements =
```

For DB I use boolean but attributes should be string. I could have two types but doing so is fine right now.

Reply:
Problem is here:
```tsx
const slide: JSX.IntrinsicElements = <deckgo-slide-split key={uuid()} {...attributes}>
    {start}
    {end}
</deckgo-slide-split>;
```

Reply:
```
// @ts-ignore
const slide: JSX.IntrinsicElements = <deckgo-slide-split key={uuid()} {...attributes}>
    {start}
    {end}
</deckgo-slide-split>;
```

doesn't work?

Reply:
Nevermind, it does work now. Wasn't receiving your messages. Only now both of them are rendering as horizontal.

Reply:
Did you pass the `{vertical: true}` in the calling method?

Reply:
`app-create-slide.tsx`:
```tsx
<div class="item" custom-tappable onClick={() => this.addSlideSplit(SlideTemplate.SPLIT)}>
    <deckgo-slide-split class="showcase">
        <p slot="start">Content</p>
        <p slot="end">Content</p>
    </deckgo-slide-split>
</div>
<div class="item" custom-tappable onClick={() => this.addSlideSplit(SlideTemplate.SPLIT, null, {vertical: true})}>
    <deckgo-slide-split vertical={true} class="showcase">
        <p slot="start">Content</p>
        <p slot="end">Content</p>
    </deckgo-slide-split>
</div>
```

Reply:
And `addSlideSplit`?

Could you commit your modifications in your branch so I could have a look?

Reply:
Sure, can do. 

```tsx
private async addSlideSplit(template: SlideTemplate, deck?: Deck, attributes: SlideAttributes = undefined) {
    const slide: JSX.IntrinsicElements = await CreateSlidesUtils.createSlide(template, deck, this.user);
    await this.closePopover(template, slide, attributes);
}
```

Reply:
1. in `create-slides.utils.tsx` make `createSlideSplit` public

2. in `app-create-slide.tsx` modify method like the following

```
private async addSlideSplit(template: SlideTemplate, attributes: SlideAttributes = undefined) {
        const slide: JSX.IntrinsicElements = await CreateSlidesUtils.createSlideSplit(attributes);
        await this.closePopover(template, slide, attributes);
    }
```

 and of course then adapt `onClick` call